doc/use_azure_pipeline_to_run_test.md

```diff
@@ -0,0 +1,43 @@
+# Use Azure Pipelines to Run Tests
```

Comment:
thinking should this doc be public in github, as it's used for our daily development and the pipeline is inside our org.

Reply:
Yes, it should be.